test/e2e/lib/hooks/index.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Internal dependencies
+ */
+import { startVideo, stopVideo, takeScreenshot } from './video-recorder';
+
+export const mochaHooks = {
+	afterAll: [ stopVideo ],
+	beforeAll: [ startVideo ],
+	afterEach: [ takeScreenshot ],
```

Comment:
The previous behavior stopped the video recorder after a failing test. Now we only stop at the end of the entire suite (?) -- does that make a difference in functionality?

Reply:
In practice, it doesn't. We only run one suite per process, so effectively, a failing test is the end of the suite.